src/EthPool.sol

```diff
@@ -89,6 +89,13 @@ contract EthPool is Pool {
             );
     }
 
+    /// @notice Gives funds from the sender of the message to the receiver.
+    /// The receiver can collect them immediately.
+    /// @param receiver The receiver
+    function give(address receiver) public payable {
+        _giveInternal(msg.sender, receiver, uint128(msg.value));
```

Comment:
I think this line is missing?
`_transferToContract(msg.sender, uint128(msg.value));`

Reply:
Outdated